packages/internal/test-util/src/lib/noop-driver/noop-driver.module.ts

```diff
@@ -11,15 +13,20 @@ import { NoopDriverConfig } from './noop-driver.config';
  */
 @NgModule()
 export class NoopDriverModule {
-  static forRoot(config?: NoopDriverConfig): ModuleWithProviders<NoopDriverRootModule> {
+  static forRoot(config: Partial<NoopDriverConfig> = {}): ModuleWithProviders<NoopDriverRootModule> {
     return {
-      ngModule: NoopDriverRootModule,
       providers: [
         {
           provide: noopDriverConfigToken,
-          useValue: config || {},
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({
```

Comment:
suggestion: add explicit return type to avoid type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): LumberjackLogDriverConfig => ({
```